### PR TITLE
feat: ignore endpoints marked as hidden

### PR DESCRIPTION
**Pull request by nanndoj**
### Background
Endpoints marked as hidden are still rendered by this library.

```
 @Post("/hidden")
  @MulterOptions({ dest: "" })
  @Hidden()
  private hiddenApi(
    @MultipartFile("hiddenParam") file: PlatformMulterFile
  ): string {
    return "";
  }
```

The above example should not be rendered because it is marked as hidden

### CODE REVIEW

File: services/ClientApiService.spec.ts [hunk 1]
```diff
@@ -37,6 +40,21 @@ export class TestCtrl {
   ): string {
     return "";
   }
+
+  @Post("/file")
```
Comment (kristelviilup) [inline services/ClientApiService.spec.ts]: what is this file endpoint?

**Review by kristelviilup — CHANGES_REQUESTED** [review]

File: services/ClientApiService.spec.ts (hunk 1 above)
Comment (nanndoj) [reply services/ClientApiService.spec.ts]: Ops! This was a mistake. Removing...

**Review by kristelviilup — APPROVED** [review]
Looks good and tests were also updated. Approved PR.

**Pull request merged**